tree/ntuple/v7/inc/ROOT/RPageStorage.hxx

```diff
@@ -186,6 +204,11 @@ public:
    virtual RPage PopulatePage(ColumnHandle_t columnHandle, NTupleSize_t globalIndex) = 0;
    /// Another version of PopulatePage that allows to specify cluster-relative indexes
    virtual RPage PopulatePage(ColumnHandle_t columnHandle, const RClusterIndex &clusterIndex) = 0;
+
+   /// Populates all the pages of the given cluster id and columns; some of the columns might not contain
+   /// any pages, but the corresponding column ids must be registered in the cluster nevertheless.
```

Comment:
Why must the column ids be registered nonetheless?

Reply:
Clarified the comment. If columns are not marked as "present", next time around when the cluster is requested, the cluster pool would think there are missing columns and re-trigger `LoadCluster()`